_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
+
+  - GeoTIFF format stores a **broad range of georeferencing information**, catering to geographic
+  as well as projected coordinate systems.
+
+> ## Why GeoTIFF format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the GeoTIFF format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the GeoTIFF format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that GeoTIFF format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> GeoTIFF format is independent of computer’s architecture, operating system, and graphics hardware.
+>> Also, the GeoTIFF format can contain both data and geographic information.
+>>
+>> 2. Therefore, yes, the GeoTIFF format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
 
- Common file format (FAIR or unFAIR??)
+- [HDF5](https://support.hdfgroup.org/HDF5/doc1.6/UG/03_Model.html):
+The Hierarchical Data Format Version 5, (HDF5) implements a model for managing and storing data,
+developed by the National Center for Supercomputing Applications (NCSA):
+![HDF5]({{ page.root }}/fig/hdf5_data_model.png)
 
-netcdf_data_model
+  - HDF5 is a **general-purpose**, **machine-independent** standard for storing scientific data in files.
 
-GRIB Format
+  - HDF5 files are organized in a hierarchical structure, with two primary structures: groups and datasets.
 
-GeoTIFF
+  - HDF5 dataset: a multidimensional array of **data** elements, together with supporting **metadata**.
 
-AAIGRID
+  - There are four essential classes of metadata: name, datatype, dataspace, and storage layout.
 
-HDF5_data_model
+  - An HDF5 structure is **self-describing**, allowing an application to interpret
+  the structure and contents of a file without any outside information.
 
-netcdf-zarr-data-model
+> ## Why HDF5 format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the HDF5 format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the HDF5 format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that HDF5 format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> HDF5 is a general-purpose, machine-independent format.
+>> Also, HDF5 format can contain both data and metadata.
+>>
+>> 2. Therefore, yes, HDF5 format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
 
-Vector data (e.g. shapefiles)
+> ## Other formats for geospatial data
+>
+> There are many other formats to store geospatial data like
+> [SHP](https://en.wikipedia.org/wiki/Shapefile)(shapefiles) for vector data,
+> [DBF](https://en.wikipedia.org/wiki/.dbf)(database file), and
+> [NetCDF ZARR Data](https://www.unidata.ucar.edu/blogs/developer/en/entry/netcdf-zarr-data-model-specification), etc.
```

Comment:
Should we add an exercise to repeat the above questions (why did you choose this, does it help to make FAIR) for their own datasets they chose above?

Reply:
If I understood it correctly, at the top of the lesson, there is a discussion "Common file format for geospatial data in your community/research team". I am wondering if it addresses your concern.